### ISSUE TITLE
Poor performance of discriminator on different GAN

### ISSUE THREAD

**Issue by mchong6** [autonomousvision/stylegan_xl#65]
I am using a different GAN repo (specifically rosalinity stylegan2 repo) and I intend to use the projected discriminator in this repo instead. I am generating faces at 256 resolution, in the range of [-1, 1] and these are my hyper params for the discriminator

`discriminator = ProjectedDiscriminator(
    backbones=['deit_base_distilled_patch16_224', 'tf_efficientnet_lite0'],
    diffaug=True,
    interp224=True,
    cout=64,
    expand = True,
    proj_type = 2,
    num_discs = 4,
    cond=False
).to(device).train()

discriminator.feature_networks.requires_grad_(False)

d_optim = optim.Adam(
    discriminator.parameters(),
    lr=2e-3,
    betas=(0., 0.99),
)
`

I did not use the discriminator image blurring trick described in the paper, I use standard non saturating GAN loss with no regularization.However the training crashes from the very beginning.
<img width="620" alt="image" src="https://user-images.githubusercontent.com/20831135/172692815-5fbee719-0a70-4131-ae22-4258da713e6e.png">

Am I missing something to make it work?

**Comment by xl-sr** [comment]
It will not work on Stylegan2 without any changes. My suggestions are to use blurring , adjust the learning rate, and use the StyleGAN 1 architecture without skip connections. Also, hinge loss is usually better.
